lib/DiscordMusicBot.js

```diff
@@ -101,6 +101,8 @@ class DiscordMusicBot extends Client {
       this.warn("Only enable this if you know what you are doing!");
       process.on("unhandledRejection", (error) => console.log(error));
       process.on("uncaughtException", (error) => console.log(error));
+      process.on("warning", (error) => console.log(error));
+			process.on("error", (error) => console.log(error));
```

Comment:
There is no process event called `error`

Reply:
Hmm how could this thing will make everything is verbose in console

Reply:
Ye I agree there's no event called `error`

Reply:
Hmm.. It worked for me.

Reply:
Ah, it was the unhandled rejection. I guess the warning would be an addition.